tests/test_myt.py

```diff
@@ -54,7 +54,7 @@ def _load_from_file(self, filename: str):
         with open(filename, encoding="UTF-8") as json_file:
             return json.load(json_file)
 
-    async def request(
+    async def request(  # pylint: disable=R0915; # noqa
```

Comment:
Please provide a explanation for why you want to disable pylint here

Reply:
I had to add two more regex matches to the tests and now it complains about it being too long